nixos/tests/rsyslogd.nix

```diff
@@ -0,0 +1,27 @@
+import ./make-test.nix ({ pkgs, lib, ... }:
```

Comment:
tests that are supposed to run on hydra are also added to `nixos/release.nix`.

Reply:
I have added the test to nixos/release.nix.